### DISCUSSION TITLE
Discussion topics for Feb 14, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4340] [Show and tell]
Discussion topics for Feb 14, 2022 Engineering Sync

**Comment by jimduff-usds** [comment]
Further discussion on release notes.
- Our auto-generated change log is here:  https://github.com/CDCgov/prime-reportstream/releases

- Our new consolidated human-generated release notes is here:  https://github.com/CDCgov/prime-reportstream/blob/d8fd33ee09286ed180ccf439de440f7e41494d83/prime-router/docs/release-notes.md

**Comment by carlosfelix2** [comment]
@carlosfelix2 What happened to Docker Desktop license issue?
